### PR TITLE
gave heads more swag in their locker (mainly CMO)

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

gives CE combat gloves instead of insuls, CMO gets the military style breath mask, an advanced medkit and a chance at a combat medkit

all heads also get oxygen/nitrogen tanks in their lockers that have their hardsuits inside, except for hos, they get a double emergency oxygen tank (so they can have that in their pocket and their WT550 in suit storage)
all heads get a chance to get a figurine of their employees

paramedic gets an emergency crowbar, double emergency oxygen tank and a combat medkit

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: JoeHammad
- add: Due to the head of security showing restraint when given the chance to execute the clown, centcom has rewarded the station heads by giving them little trinkets in their lockers.. sometimes

### CODE REVIEW

**Comment by JoeHammad1844** [comment]
hm. too much in hos's locker

**Comment by metalgearsloth** [comment]
Seems like bloat, heads don't need a full set of gas tanks / toys in their lockers.

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml
Comment (ficcialfaint) [inline Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:1]: `LockerHeadOfSecurityFilledHardsuit` has too many items (two need to be removed)

**Review by ficcialfaint — CHANGES_REQUESTED** [review]

**Comment by JoeHammad1844** [comment]
> Seems like bloat, heads don't need a full set of gas tanks / toys in their lockers.

its for when the ones with hardsuits inside only, meaning they have no suit storage locker, meaning they need to scavenge or ask atmos for oxygen, which is annoying

**Comment by ficcialfaint** [comment]
I'd leave the lockers with oxygen only tanks (without nitrogen one), and to me it's kinda weird that a head of a department has to ask anyone else to be able to survive emergency situations

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Put the oxy tank in the hardsuit locker, don't make the other changes.

**Comment by JoeHammad1844** [comment]
damn, really?

**Comment by JoeHammad1844** [comment]
is that alright now?

**Comment by JoeHammad1844** [comment]
by the way i'm gonna be gone for around 5 days on holiday tomorrow

**Comment by metalgearsloth** [comment]
I had a chat about it with a few maints; if you add too much stuff like toys or whatever it loses their novelty and detracts from what the locker should actually be doing i.e. providing a spare set of gear if the role doesn't spawn / any other unique role essentials.

If something can be grabbed from elsewhere (e.g. gas tanks) then it should be.

**Comment by JoeHammad1844** [comment]
alright then, makes sense

**Comment by JoeHammad1844** [comment]
anything else?

**Comment by Emisse** [comment]
still too much bloat

File: Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml [hunk 1]
```diff
@@ -197,8 +197,7 @@
   components:
   - type: StorageFill
     contents:
-        - id: NitrogenTankFilled
-        - id: OxygenTankFilled
+        - id: DoubleEmergencyOxygenTankFilled
```
Comment (Partmedia) [inline Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml:200]: Is nitrogen just not going to be available from suit storage?

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml [hunk 2]
```diff
@@ -134,15 +142,18 @@
   - type: StorageFill
     contents:
       - id: ClothingOuterHardsuitEngineeringWhite
+      - id: OxygenTankFilled
```
Comment (Partmedia) [inline Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:145]: You're duplicating a oxygen tank right below.

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml [hunk 3]
```diff
@@ -134,15 +142,18 @@
...
       - id: ClothingMaskBreath
       - id: OxygenTankFilled
-      - id: NitrogenTankFilled
```
Comment (Partmedia) [inline Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:139]: Why is nitrogen being removed?

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml [hunk 4]
```diff
@@ -42,8 +44,11 @@
       - id: ClothingHeadHatCaptain
       - id: ClothingNeckCloakCap
       - id: ClothingHandsGlovesCaptain
+      - id: ToyFigurineCaptain
+        prob: 0.15
       - id: ClothingOuterHardsuitCap
       - id: ClothingMaskGasCaptain
+      - id: OxygenTankFilled
```
Comment (Partmedia) [inline Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:51]: Missing nitrogen. Alternatively, why can't the captain visit engineering for a tank?

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml [hunk 5]
```diff
@@ -329,6 +360,8 @@
       - id: ClothingHeadHatHoshat
       - id: ClothingNeckCloakHos
       - id: ClothingOuterCoatHoSTrench
+      - id: SadTromboneImplant
```
Comment (Partmedia) [inline Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:363]: This doesn't seem necessary.

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml (hunk 2 above)
Comment (JoeHammad1844) [reply Resources/Prototypes/Catalog/Fills/Lockers/heads.yml]: resolved

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml (hunk 5 above)
Comment (JoeHammad1844) [reply Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:363]: neither does the shark plushie

Comment (metalgearsloth) [reply Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:363]: It got removed

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml (hunk 4 above)
Comment (JoeHammad1844) [reply Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:51]: I did add it, but I was told to remov e it by metal gear sloth

**Comment by Fahasor** [comment]
#17966 may be resolved

**Comment by JoeHammad1844** [comment]
hmm im gonna need to discard the commits to resolve the conflicts, please stand by

**Comment by Partmedia** [comment]
Sorry to have kept you working on this pull request. I think I'm concerned that the heads' lockers already contain a lot of stuff, and that the additions proposed in this PR aren't adding a lot of game play value to these heads. Besides the gas tanks and perhaps the medkit changes, I don't think we're looking to add much more to the heads' lockers.

If you'd like, we can discuss more on Discord; or I'll go ahead and work on a modified version of this PR with just the gas tanks.

**Comment by JoeHammad1844** [comment]
image
hmm

**Comment by JoeHammad1844** [comment]
hmm, maybe I should close this and reduce the scope

**Pull request closed without merging**